pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -58,6 +66,14 @@ const (
 	SubnetTopologyPrivate SubnetTopology = "Private"
 	// SubnetTopologyPublic represents publicly-routed subnets
 	SubnetTopologyPublic SubnetTopology = "Public"
+
+)
+
+var (
+	// True holds true value and can have it's address taken
+	True = true
```

Comment:
usages of this should be replaced by `Enabled()`

Reply:
Replaced all uses of False/True with Disabled()/Enabled()